src/components/Editor/SearchBar.js

```diff
@@ -68,7 +68,7 @@ class SearchBar extends Component {
     highlightLineRange: ({ start: number, end: number }) => void,
     clearHighlightLineRange: () => void,
     searchOn?: boolean,
-    toggleActiveSearch: (?ActiveSearchType) => any,
```

Comment:
switching from the verb toggle to set because we have more than two now: Project, source, file, symbol